OpenAISharp.Completion/OpenAISharp.Completion.csproj

```diff
@@ -2,6 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageId>CodeMonarchs.OpenAISharp.Completion</PackageId>
+    <Description>Wrapper for the OpenAI Completion API</Description>
+    <Copyright>Copyright © CodeMonarchs</Copyright>
```

Comment:
Not sure if I picked the right license for this repo, I picked Public GNU (but maybe it's MIT?),  not sure.